src/components/Nav/Nav.tsx

```diff
@@ -111,32 +161,37 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
 
   private _renderGroup(group: INavLinkGroup, groupIndex: number): React.ReactElement<{}> {
     const isGroupExpanded: boolean = this.state.isGroupExpanded[groupIndex] !== false;
-    const hasGroupButton: boolean = !!(group.name);
 
     return (
       <div key={ groupIndex } className={ css('ms-Nav-group', { 'is-expanded' : isGroupExpanded }) }>
-        { (hasGroupButton ?
+        { (group.name ?
         <button
-          className='ms-Nav-groupButton'
+          className='ms-Nav-chevronButton ms-Nav-chevronButton--group'
```

Comment:
these icon names robalby have changed. Please see the latest. Notice below you're undoing some changes like ChevronDown